sdk/spring/spring-cloud-azure-autoconfigure/pom.xml

```diff
@@ -278,6 +278,14 @@
       <version>2.0.2</version> <!-- {x-version-update;jakarta.validation:jakarta.validation-api;external_dependency} -->
     </dependency>
 
+    <!-- jdbc   -->
+    <dependency>
+      <groupId>com.microsoft.sqlserver</groupId>
+      <artifactId>mssql-jdbc</artifactId>
+      <version>10.2.0.jre11</version> <!-- {x-version-update;com.microsoft.sqlserver:mssql-jdbc;external_dependency} -->
+      <scope>provided</scope>
+    </dependency>
+
```

Comment:
If we are not going to support mssql this time, should we remove this?

Reply:
Fixed when resolving git conflicts.